runtimes/runtimes/standalone.ts

```diff
@@ -72,6 +72,11 @@ if (checkAWSConfigFile()) {
     process.env.AWS_SDK_LOAD_CONFIG = '1'
 }
 
+process.on('uncaughtException', err => {
+    console.error('Uncaught Exception:', err.message)
+    process.exit(1)
```

Comment:
This nodejs feature is quasi-deprecated, and newer node versions have CLI flags that control how to handle uncaught exceptions/rejections.

- `--unhandled-rejections=mode` https://nodejs.org/api/cli.html#--unhandled-rejectionsmode
- `--abort-on-uncaught-exception` https://nodejs.org/api/cli.html#--abort-on-uncaught-exception

Also note that any code loaded in this nodejs process can overwrite this `on('uncaughtException')` handler (accidentally).